packages/thirdweb/src/wallets/coinbase/coinbaseWebSDK.ts

```diff
@@ -290,12 +302,43 @@
             accessList: tx.accessList,
             value: tx.value ? numberToHex(tx.value) : undefined,
             gas: tx.gas ? numberToHex(tx.gas) : undefined,
-            from: this.address,
+            from: getAddress(address),
             to: tx.to as Address,
             data: tx.data,
           },
         ],
       })) as Hex;
+
+      // Non-blocking wait for receipt to track transaction
+      waitForReceipt({
```

Comment:
Oof, I don't think we can do this. Feels very heavy to always poll just for a tracking event.

Can't we just track on submit on chain? We have the tx hash here, why fo we need to wait for confirmation?

Reply:
Yeah that's fair, we just can't determine if it reverted or not then

Reply:
i think that's ok - we can add a separate event on transaction confirmation and correlate based on txhash. if the app cares about confirmations, then they'll wait for receipt.